app/views/auth_token/verify.html.haml

```diff
@@ -0,0 +1,7 @@
+
+.container
+
+  %section
+    .bs-callout.bs-callout-info.bs-callout-emphasize
+      Oops! We couldn't verify your token. Please ensure it's correct.
```

Comment:
We'll need to work on this error message a bit ;) It's not clear what the user should do here. I'm guessing trying to reauthenticate would be the way to go.

Reply:
agree!